### DISCUSSION TITLE
Will there be a Projectify-light?

### DISCUSSION THREAD

**Discussion by pmario** [NicolasPetton/Projectify#34] [General]
Hi, 
I've seen your version number is 0.xx.yy atm. So you are still adding functionality. 

The Homepage says: **From simple to-do lists to complete GTD workflows** ...

Do you plan to extend the App to be a full blown GTD like: https://dyumnin.com/Cardo.html ?

At the moment your "internal connections" seems to be based on "tags only", which I like a lot for "TW newbie" users. ... Do you intend to keep it that way?

I personally think, as soon as you start to use fields to manage your internal connections, the project starts to get heavy weight. ... IMO that would be the indicator for a "*-light" or "*-starter" version. ... 

Any thoughts?

**Comment by pmario** [comment]
The reason, why I ask is, that I did create a very, very basic GettingStarted-plus, which allows user to create a super-simple "task-handling". I'm using "task" instead of "todo" as a tag. 

I want to keep this GettingStarted-task-handler very basic, but I'd like to add a link to "projectify" homepage, if it stays "light weight".  

It should be possible, to export the "task" tiddlers with a "todo" tag and then import them to projectify.

The GettingStarted is German see:https://u9t.github.io/editions/MDUe_basis/  ... but the **English getting started can be imported as a plugin** and then language can be switched. [ToDo-list creation video German](https://youtu.be/YuqgGktKFNw?t=234) An English video is missing atm.

**Comment by NicolasPetton** [comment]
> Do you plan to extend the App to be a full blown GTD

With a tiny bit of extra features (mostly filtering TODOs by tags, that would act as contexts), I believe one can use Projectify already to implement the GTD methodology. I'm planning to add documentation (or maybe a tutorial) to showcase how to use it for GTD.

> At the moment your "internal connections" seems to be based on "tags only", which I like a lot for "TW newbie" users. ... Do you intend to keep it that way?

Definitely. The simplicity and ubiquity of tags make them a really good choice, I believe, for structuring data in TW. The only place where tags are not use is for project categories (a field is used instead).

> Will there be a Projectify-light?

I'm not sure what you mean with light weight, but I'm not planning to have a separate, striped-down edition of Projectify.

Projectify is made of ~150 tiddlers today (without the language pack), I'm not sure if you would consider that light weight or not :-) I'm putting a lot of effort into keeping the interface clean and simple, maybe that's what you mean by light weight?

Regarding features, as I envision it, we're 99% there. The only really missing bits are:
- support for recurring tasks
- support for filtering todos by tags

Now Projectify is indeed evolving quickly, and I keep adding new features based on the feedback I receive, but I will always keep it "simple", meaning that I will always be careful when introducing new user-visible elements and will keep paying attention to the UX and UI.

Cheers!
Nico

**Comment by pmario** [comment]
>>Do you plan to extend the App to be a full blown GTD
>
>With a tiny bit of extra features (mostly filtering TODOs by tags, that would act as contexts), I believe one can use Projectify already to implement the GTD methodology. I'm planning to add documentation (or maybe a tutorial) to showcase how to use it for GTD.

That's good. The only thing you should take care is: If you use the term: GTD and / or "Getting Things Done", you need to make it clear, that your project has no relation to the David Allen Company. see an example like: http://gsd5.tiddlyspot.com/#Disclaimer

GTD and "Getting Things Done" are a registered trademarks and they did react prompt to other TW projects, that got some traction

**Comment by pmario** [comment]
>Definitely. The simplicity and ubiquity of tags make them a really good choice, I believe, for structuring data in TW. The only place where tags are not use is for project categories (a field is used instead).

That's good.

**Comment by pmario** [comment]
>...Projectify is made of ~150 tiddlers today (without the language pack), I'm not sure if you would consider that light weight or not :-) I'm putting a lot of effort into keeping the interface clean and simple, maybe that's what you mean by light weight?

I would consider that as light weight, for a full blown edition.

**Comment by pmario** [comment]
>Regarding features, as I envision it, we're 99% there. The only really missing bits are:
>
> - support for recurring tasks
> - support for filtering todos by tags

Cool!
